addons/board/static/src/scss/dashboard.scss

```diff
@@ -123,8 +123,8 @@
             }
 
             // Override height of graph. min-height doesn't do the trick
-            .o_graph {
-                height: 500px;
+            .o_graph_svg_container {
+                height: 250px;
```

Comment:
Use the same variable in board and web_dashboard

Reply:
@adr-odoo 
Did you mean putting the child operator ">", or copying the whole style from web_dashboard ?

Reply:
Non, juste avoir une variable avec 250px et l'utiliser des deux cotés.